analysed-packages/orc/version-0.4.32/orc-0.4.32-SPDX2TV.spdx

```diff
@@ -0,0 +1,2089 @@
+SPDXVersion: SPDX-2.3
+DataLicense: CC0-1.0
+
+##-------------------------
+## Document Information
+##-------------------------
+
+DocumentNamespace: http://fossology-6674c79fbf-2qzn2/repo/SPDX2TV_orc-0.4.32.tar.xz_1717056492.spdx
+DocumentName: /srv/fossology/repository/report
+SPDXID: SPDXRef-DOCUMENT
+
+##-------------------------
+## Creation Information
+##-------------------------
+
+Creator: Tool: fossology-4.4.0
+Creator: Person: Sascha Dierberg
+CreatorComment: <text>
+This document was created using license information and a generator from Fossology.
+It contains the license and copyright analysis of orc 0.4.32.
+Please check "LicenseComments" for explanations of concluded licenses.
+</text>
+Created: 2024-05-30T08:08:12Z
+LicenseListVersion: 3.22
+
+##-------------------------
+## Package Information
+##-------------------------
+
+
+PackageName: orc-0.4.32.tar.xz
+PackageFileName: orc-0.4.32.tar.xz
+SPDXID: SPDXRef-upload149
+PackageDownloadLocation: NOASSERTION
+FilesAnalyzed: true
+PackageVerificationCode: b5fcf66a5830bfd349537a74ba49761bcea8990f
+PackageChecksum: SHA1: 1a0ff7f917913d6fabbc9ecea56496361ff7f3bf
+PackageChecksum: SHA256: a66e3d8f2b7e65178d786a01ef61f2a0a0b4d0b8370de7ce134ba73da4af18f0
+PackageChecksum: MD5: 4fd61be43bb222fac1ecc486f4d055db
+PackageLicenseConcluded: NOASSERTION
+PackageLicenseDeclared: LicenseRef-BSD-2-Clause
```

Comment:
according to the information in the COPYING file (see below) the PackageLicenseDeclared shall be LicenseRef-BSD-2-Clause-5981c42271f3da678592764a0221eb97 instead of LicenseRef-BSD-2-Clause. As you have correctly concluded for the individual files. Perhaps this is a current short coming of the particular FOSSology version you are using.
Can you please correct the PackageLicenseDeclared?

COPYING:
The majority of the source code and the collective work is subject
to the following license:

...

Redistribution and use in source and binary forms, with or without
modification, are permitted provided that the following conditions
are met:
1. Redistributions of source code must retain the above copyright
notice, this list of conditions and the following disclaimer.
2. Redistributions in binary form must reproduce the above copyright
notice, this list of conditions and the following disclaimer in the
documentation and/or other materials provided with the distribution.

THIS SOFTWARE IS PROVIDED BY THE AUTHOR ``AS IS'' AND ANY EXPRESS OR
IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
ARE DISCLAIMED. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR ANY DIRECT,
INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
(INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION)
HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT,
STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING
IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
POSSIBILITY OF SUCH DAMAGE.